tensorrt_llm/_torch/custom_ops/torch_custom_ops.py

```
@@ -1308,15 +1243,15 @@ def w4a8_mxfp4_fp8_gemm(
-        output_dtype: torch.dtype,
-        output_buffer_kind: int = int(BufferKind.DEFAULT),
+    act_fp4: torch.Tensor,
+    weight: torch.Tensor,
+    act_sf: torch.Tensor,
+    weight_scale: torch.Tensor,
+    alpha: torch.Tensor,
+    output_dtype: torch.dtype,
+    to_userbuffers: bool = False,
 ) -> torch.Tensor:
-    return act_fp4.new_empty((act_fp4.size(0), weight.size(0)),
+    return act_fp8.new_empty((act_fp8.size(0), weight.size(0)),
```

Flake8 (7.3.0)

[error] undefined name 'act_fp8'

(F821)